sdk/highlight-run/src/client/utils/randomUuidV4.ts

```diff
@@ -0,0 +1,107 @@
+// This implementation is the same as in the browser package. Eventually we
```

Comment:
So basically this is a compliant UUID v4 implementation which I had implemented for browser-telemetry. It was available, so I copy pasted it over here. In the vast majority of cases it uses browser randomUUID and only uses the implementation when it isn't available.

If this it too much we could:
1. Use a basic, but not really compliant implementation as is used in the unit tests. (Or maybe one similar to the OTEL one).
2. Depend on crypto.randomUUID. Not sure if the SDK is compatible with SDKs that don't have it.
3. 2 + let it polyfill.